hack/release.sh

```diff
@@ -91,7 +91,7 @@ ALLOW_OVERRIDE=${ALLOW_OVERRIDE:-}
 SKIP_BUILD=${SKIP_BUILD:-}
 SKIP_PUSH_LATEST=${SKIP_PUSH_LATEST:-}
 LINUX_ARCH=${LINUX_ARCH:-amd64 arm arm64 ppc64le s390x}
-WINDOWS_DISTROS=${WINDOWS_DISTROS:-ltsc2019 1909 2004 20H2}
+WINDOWS_DISTROS=${WINDOWS_DISTROS:-ltsc2019 1909 2004 20H2 ltsc2022}
```

Comment:
I think SAC 1909 and 2004 is end of service.

Reply:
great, I’m going to remove them